src/components/Header.js

```diff
@@ -90,6 +92,17 @@ const Header = ({ routes = [{ name: 'home', path: '/' }] }) => {
                   </Button>
                 )
             )}
+            {
+              user.isLogged === true?
+              <Stack direction="row" spacing={2}>
+                <Avatar alt={`user ${user.data.id} image`} src={user.data.image} />
+              </Stack>:
+              <Button href='/login'>
```

Comment:
Debería ser `<Button component={Link} to='/login'>` para usar enlace de react-router y no salir de la SPA